src/components/membership-cache.ts

```diff
@@ -103,7 +103,7 @@ export class MembershipCache {
             return Object.keys(this.membershipMap[roomId]);
         }
         const members = [];
-        for (const [userId, [membership, profile]] of Object.entries(this.membershipMap[roomId])) {
```

Comment:
Random linter error I spotted

Reply:
Why was this an error?
The value of `profile` isn't used.

Reply:
Well, warning. The linter wasn't happy that profile wasn't used.

Reply:
I also read this the commit wrong way, thinking that `profile` got added.
Nevermind. :see_no_evil: